hypertrace-ingester/src/test/java/org/hypertrace/migration/MigrationTestRpc.java

```diff
@@ -0,0 +1,335 @@
+import org.junit.jupiter.params.provider.MethodSource;
+
+public class MigrationTestRpc {
+
+  private final Random random = new Random();
+  private JaegerSpanNormalizer normalizer;
+
+  @BeforeEach
+  public void setup()
+      throws SecurityException, NoSuchFieldException, IllegalArgumentException,
+          IllegalAccessException {
+    // Since JaegerToRawSpanConverter is a singleton, we need to reset it for unit tests to
+    // recreate the instance.
+    Field instance = JaegerSpanNormalizer.class.getDeclaredField("INSTANCE");
+    instance.setAccessible(true);
+    instance.set(null, null);
+
+    // Create a JaegerSpanNormaliser
+    String tenantId = "tenant-" + this.random.nextLong();
+    Map<String, Object> configs = Map.of("processor", Map.of("defaultTenantId", tenantId));
+    this.normalizer = JaegerSpanNormalizer.get(ConfigFactory.parseMap(configs));
+  }
+
+  @Test
+  public void testGrpcFields() throws Exception {
+
+    Map<String, String> tagsMap =
+        new HashMap<>() {
+          {
+            put(RawSpanConstants.getValue(GRPC_ERROR_MESSAGE), "Some error message");
+            put(RawSpanConstants.getValue(CENSUS_RESPONSE_STATUS_CODE), "12");
+            put(RawSpanConstants.getValue(GRPC_STATUS_CODE), "13");
+            put(RawSpanConstants.getValue(CENSUS_RESPONSE_CENSUS_STATUS_CODE), "14");
+            put(
+                RawSpanConstants.getValue(CENSUS_RESPONSE_STATUS_MESSAGE),
+                "CENSUS_RESPONSE_STATUS_MESSAGE");
+            put(RawSpanConstants.getValue(ENVOY_GRPC_STATUS_MESSAGE), "ENVOY_GRPC_STATUS_MESSAGE");
+            put(RawSpanConstants.getValue(GRPC_REQUEST_BODY), "some grpc request body");
+            put(RawSpanConstants.getValue(GRPC_RESPONSE_BODY), "some grpc response body");
+          }
+        };
+
+    Span span = createSpanFromTags(tagsMap);
+    RawSpan rawSpan = normalizer.convert("tenant-key", span);
+
+    assertAll(
+        () ->
+            assertEquals(
+                rawSpan.getEvent().getGrpc().getResponse().getErrorMessage(),
+                RpcSemanticConventionUtils.getGrpcErrorMsg(rawSpan.getEvent())),
+        () ->
+            assertEquals(
+                rawSpan.getEvent().getGrpc().getResponse().getStatusCode(),
+                RpcSemanticConventionUtils.getGrpcStatusCode(rawSpan.getEvent())),
+        () ->
+            assertEquals(
+                rawSpan.getEvent().getGrpc().getResponse().getStatusMessage(),
+                RpcSemanticConventionUtils.getGrpcStatusMsg(rawSpan.getEvent())),
+        () ->
+            assertEquals(
+                rawSpan.getEvent().getGrpc().getResponse().getSize(),
+                RpcSemanticConventionUtils.getGrpcResponseSize(rawSpan.getEvent()).get()),
+        () ->
+            assertEquals(
+                rawSpan.getEvent().getGrpc().getRequest().getSize(),
+                RpcSemanticConventionUtils.getGrpcRequestSize(rawSpan.getEvent()).get()));
```

Comment:
Same comment as above (https://github.com/hypertrace/hypertrace-ingester/pull/214/files#r664253100) for `getGrpcRequestSize` method, remove the if condition wrapping `if (attributeValueMap.get(RawSpanConstants.getValue(GRPC_REQUEST_BODY)) != null)`